### ISSUE TITLE
ORTC Demo: labels are not associated with form controls

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#240]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/ortcdemo/

The labels such as Your ID are not associated with their control. They need to have a for attribute with the value as the ID of the control below.

**Comment by alrra** [comment]
@dstorey @sarvaje This demo is external, namely, it's from https://ortcdemo.azurewebsites.net.

**Comment by alrra** [comment]
@dstorey @sarvaje  See @molant's [comment](https://github.com/MicrosoftEdge/Demos/issues/196#issuecomment-267626168)

**Comment by ShijunS** [comment]
@molant will help fix the issue.  Thanks!

**Comment by molant** [comment]
This is fixed now.
